edgelet/kube-client/src/client.rs

```diff
@@ -77,6 +78,58 @@ where
     S::Error: Fail,
     Body: From<<S as Service>::ResBody>,
 {
+    pub fn is_subject_allowed(
+        &mut self,
+        resource: Option<String>,
+        verb: Option<String>,
+    ) -> impl Future<Item = bool, Error = Error> {
```

Comment:
I noticed that all the rest methods return k8s objects, but this one returns `bool`. Does k8s payload provide additional info that can be helpfull later on?

Reply:
Doesn't hurt to return the SubjectAccessReviewStatus struct.